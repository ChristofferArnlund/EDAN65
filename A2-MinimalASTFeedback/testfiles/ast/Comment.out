Program
  List
    Func
      IdDecl
        Name ID="a"
        Opt
      List
      StmtBlock
        List
          IdDecl
            Name ID="x"
            Opt
          IdDecl
            Name ID="y"
            Opt
          Return
            Numeral NUMERAL="0"
