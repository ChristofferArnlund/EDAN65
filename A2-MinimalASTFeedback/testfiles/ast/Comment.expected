Program
  List
    Func
      Name ID="a"
      List
      List
        IdDecl
          Name ID="x"
          Opt
        IdDecl
          Name ID="y"
          Opt
        Return
          Numeral NUMERAL="0"
